src/components/ProfileStepper.jsx

```diff
@@ -9,8 +9,7 @@ import StepLabel from '@material-ui/core/StepLabel';
 import StepContent from '@material-ui/core/StepContent';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
-import ProfileInput from './ProfileInput';
-import ScholarshipInput from './ScholarshipInput';
+import ProfileInput from './ProfileForm';
```

Comment:
Name is inconsistent

Reply:
No longer an issue the only import is `ProfileForm`, which now is consistent with component name and file name along with the function name